src/messages/imageloadermanager.hpp

```diff
@@ -15,6 +15,8 @@ class ImageLoaderWorker : public QObject
 public slots:
     void handleRequest(chatterino::messages::LazyLoadedImage *lli, QNetworkAccessManager *nam);
     void handleLoad(LazyLoadedImage *lli, QNetworkReply *reply);
+signals:
```

Comment:
I think clang-format needs to be run here, or just add an empty line above `signals:`